config/application.rb

```diff
@@ -34,7 +33,7 @@ class Application < Rails::Application
 
     # Changes in rails 7.0.3.1 prevented Symbols being used in serialised fields
     # this overrides the setting and allows the code(and tests) to run as normal
-    config.active_record.yaml_column_permitted_classes = [Symbol, Date, Remarks]
+    config.active_record.yaml_column_permitted_classes = [Symbol, Date]
```

Comment:
can this line not be deleted now?

Reply:
@starswan Not sure why this line was added. Lets remove it separately as "no ticket" PR

Reply:
It was added because we were serializing the remarks object - which we're no longer doing in this PR, so it can go back to defaults IMHO